### PR TITLE
Fix zmon resource limits

### PR DESCRIPTION
**Pull request by mohabusama**
**Changes**

- Increase zmon-scheduler memory requests since it is constantly restarting
- Adjust other components based on observed 7-Days metrics

### CODE REVIEW

**Comment by mohabusama** [comment]
@aermakov-zalando I saw the changes merged earlier after pushing my changes. The current adjustment are based on metrics observed on most clusters.

File: cluster/manifests/zmon-scheduler/deployment.yaml [hunk 1]
```diff
@@ -46,10 +46,11 @@ spec:
           image: "registry.opensource.zalan.do/stups/zmon-scheduler:v43"
           resources:
             limits:
-              memory: 1.5Gi
+              cpu: 1
+              memory: 2.5Gi
             requests:
-              cpu: 250m
-              memory: 1.5Gi
+              cpu: 1
```
Comment (linki) [inline cluster/manifests/zmon-scheduler/deployment.yaml]: Should we rather use `1000m` here?

Comment (mohabusama) [reply cluster/manifests/zmon-scheduler/deployment.yaml]: Yep sure.

**Comment by aermakov-zalando** [comment]
Can you move `logging-agent` changes to a separate PR? Changing requests on daemonset pods is tricky. :(

**Comment by mohabusama** [comment]
@aermakov-zalando yep, no problem

**Comment by aermakov-zalando** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**